src/gl/opengl/shaders/mobile/atmosphereVertexShader.vert

```diff
@@ -0,0 +1,21 @@
+#version 300 es
+
+layout(std140) uniform type_u_vertParams
+{
+    layout(row_major) mat4 u_modelFromView;
+    layout(row_major) mat4 u_viewFromClip;
+} u_vertParams;
+
+layout(location = 0) in vec3 in_var_POSITION;
+layout(location = 1) in vec2 in_var_TEXCOORD0;
+out vec2 out_var_TEXCOORD0;
```

Comment:
You need to rerun the shader generation, the script was modified to make these `varying_` instead of `out_var_`/`in_var_` for lines that don't have `layout(…)`.